src/current/_includes/v23.1/sidebar-data/cloud-deployments.json

```diff
@@ -131,28 +131,28 @@
             ]
             },
             {
-            "title": "CMEK for Dedicated",
+            "title": "Customer-Managed Encryption Keys (CMEK)",
             "items": [
                 {
-                "title": "Manage Customer-Managed Encryption Keys (CMEK) for Dedicated",
+                "title": "Manage CMEK for Dedicated",
                 "urls": [
                     "/cockroachcloud/managing-cmek.html"
                 ]
                 },
                 {
-                "title": "Provision AWS KMS Keys and IAM Roles for CMEK",
+                "title": "Provision AWS for CMEK",
                 "urls": [
                     "/cockroachcloud/cmek-ops-aws.html"
                 ]
                 },
                 {
-                "title": "Provision GCP KMS Keys and Service Accounts for CMEK",
+                "title": "Provision GCP for CMEK",
                 "urls": [
                     "/cockroachcloud/cmek-ops-gcp.html"
                 ]
                 },
                 {
-                "title": "Customer-Managed Encryption Keys (CMEK)",
+                "title": "Customer-Managed Encryption Keys (CMEK) Overview",
```

Comment:
Generally I anticipate seeing an overview section at the top of a dropdown section (if you know what I mean), so it acts as a introduction to all the other pages. It seems like Step 3 on this page links out to the CMEK pages for GCP/AWS too. 

What do you think? Perhaps you have a reason I'm not seeing for this order!

Reply:
+1

Reply:
If we move this to the top of the CMEK subsection, I think the sidenav should just say "Overview"

Reply:
totally agree on ordering, I'll move overview to the top

@kathancox unless I'm confused, i think you might have your docs swapped, the one with Step 3 isn't the Overview page (cmek.html), its managing-cmek.htlm?